samples/bluetooth/direction_finding_central/sysbuild/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if (NOT BOARD MATCHES "nrf5340/cpuapp")
```

Comment:
this needs to be using soc Kconfig not checking a string, so yes the best place for that is in <app>/sysbuild.cmake where the sysbuild Kconfig tree has been loaded and set